docs/src/pages/components/no-ssr/FrameDeferring.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import NoSsr from '@material-ui/core/NoSsr';
+
+const useStyles = makeStyles({
+  container: {
+    maxWidth: 300,
+    wordBreak: 'break-all',
+  },
+});
+
+function LargeTree() {
+  return (
+    <>
```

Comment:
Could you rather make a typecast here (to something that's recognized by TS as a valid return type from a component) and add a comment that TS doesn't recognize arrays as valid return types (yet).

Otherwise this might be confusing in the JS demo since wrapping an array with a fragment is redundant.

Reply:
Hi @eps1lon . I have added the required changes. Please let me know if this is ok.